requirements.txt

```diff
@@ -147,7 +150,7 @@ raven==6.10.0
 recaptcha2==0.1
 redis==3.3.8
 redlock==1.2.0
-reportlab==3.1.44
+reportlab #==3.1.44
```

Comment:
This bumps it to the latest version available. I think we may want to stick with the earliest/lowest version that introduced Python3 support.

Reply:
This is merely to get `pip install` to work. Each of the commented or removed packages need to be addressed separately and not in this PR.